### PR TITLE
[DOC] Add bot api server instructions #38

### PR DESCRIPTION
**Pull request by 4TT1L4**
## Summary

Updated the README.md and added some instructions about starting the bot API server.

Related to:
- #38

## Type of Change

- Documentation update (adding or updating documentation related to the project)

## Checklist
N/A

## Testing
N/A
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,15 +162,27 @@ Endpoints made available by server are specified [here](./web/swagger/api.yaml).
...
-4. Test if server is running successfully by calling, say, `/settings` endpoint. Example `curl` request: `curl -H 'api-key: YOUR_SECRET_KEY' -X GET http://localhost:8082/v0/settings | jq`, assuming port was specified as `8082`. On success, it should return something akin to:
-    ```json
-    {
-      "network": "mainnet",
-      "version": "0.1.0",
-      "revision": "c2f8db2bc82a13c850c3b0088a1ce089bb1065b7",
-      "backend": "mmb"
-    }
+3. Run the server with command `cabal run geniusyield-server -- serve -c my-config.yaml`
+4. Run `cabal run geniusyield-server -- -h` for help 😉
```
Comment (sourabhxyz) [inline README.md:166]: This is not really the fourth step and was given as a remark for third step.

File: README.md [hunk 2]
```diff
@@ -15,9 +15,84 @@ This repository houses on-chain smart contracts, Haskell off-chain interaction l
 - [`geniusyield-server-lib`](./geniusyield-server-lib/) serves endpoints using our off-chain code to easily interact with GeniusYield DEX in language of user's choice.
 - [`geniusyield-orderbot-lib`](./geniusyield-orderbot-lib/) cater to additional requirements such as building up of an order-book, receiving price feed, etc.
 
-## Spinning up server
+# Bot API Server
```
Comment (sourabhxyz) [inline README.md:18]: Shouldn't it have `##` in prefix? Current way would also break table of contents.
Calling it "Bot ..." might not be a good idea as server could be spin up without having any signing key configured, say for a scenario of browser wallet where this API is still useful.

Comment (4TT1L4) [reply README.md:18]: I can add an extra level

File: README.md (hunk 1 above)
Comment (4TT1L4) [reply README.md:170]: I copied over this from your original docs, but I can change it if you think that it is better to have it differently.

File: README.md (hunk 2 above)
Comment (4TT1L4) [reply README.md]: I have fixed the table of contents:

image

**Pull request merged**